datastore/unit_tests/test_transaction.py

```diff
@@ -206,9 +206,12 @@ def __init__(self, xact_id=123, keys=()):
             mutation_results=mutation_results)
 
     def begin_transaction(self, project):
+        import mock
```

Comment:
This is one of the few imports that we have been putting at the top of the module because it ends up getting used a lot. I'm not sure if it matters in this case, but someone adding a test using a mock might add the import at the top of the module and then it's imported twice anyway.